soc/arm/nxp_kinetis/k6x/Kconfig.soc

```diff
@@ -117,4 +117,12 @@ config K6X_FLASH_CLOCK_DIVIDER
 	  This option specifies the divide value for the K6X flash clock from the
 	  system clock.
 
+config K6X_HSRUN
+	bool "High Speed RUN mode"
+	depends on SOC_MK66F18
+	default y
+	help
+	  This options enables support for High Speed RUN mode on K66F SoC. It is
+	  not available on K64F.
```

Comment:
Do we need to mention K64F here? The option already depends on `SOC_MK66F18`, so it will never appear in the Kconfig menus for a K64F based board.

Reply:
Ok,